opcua/server/server.py

```diff
@@ -74,7 +74,7 @@ class Server(object):
 
     """
 
-    def __init__(self, shelffile=None, iserver=None):
+    def __init__(self, shelffile=None, iserver=None, nosecurity=True):
```

Comment:
You forgot that line ;-)

Reply:
yes, now I squashed all the changes into a single commit to decrease the noise. Hope it's fine now